# Action id
int32 id

# Action "goal" is the exploration mode
int32 mode # 0 stop
           # 1 resume
           # 2 left wall
           # 3 right wall
           # 4 waypoint
           # 5 A*

# Path to follow for waypoint mode
nav_msgs/Path   path

# Should navigation target the orientation of the final goal/path
bool            targetOrientation

---

# Corresponding Action id
int32 id

# Has the mode of operation been immediately achieved.
bool success
